### ISSUE TITLE
Ability to Create Backups without Opening Explorer...

### ISSUE THREAD

**Issue by glimmerwell** [Drive4ik/simple-tab-groups#571]
Every time I launch Firefox, STG creates a backup. That is good. However; it also opens up the backup folder in Window Explorer. I am hoping there is a way to have the auto backup function, but don't automatically open Windows Explorer.

I looked in Options. I couldn't find a setting to achieve this. So I am not sure if this is a bug or a non-existing feature.

Thanks for any help!
GW

**Comment by Drive4ik** [comment]
This is something interesting!) The addon has only one place in the code where the backup folder opens - and this is the "open backup folder" button.
What version of OS, addon and browser?
Does your browser start with the system?
Does the backup folder open every time you start the browser in one OS session?

**Comment by glimmerwell** [comment]
On that computer, I have Win 10, Firefox latest (74.0 64-bit), and STG 4.5. I unchecked "enable auto backup" and the behavior seems to have stopped. But I like the idea of auto backup. 

It was doing it at least once per day, even when FF was already opened, I think. It also triggers an audio sound when the folder opens, but that may be some setting on my computer.

Let me know if I can help. 

Also, I just noticed that I can't find a way to limit how many backups STG keeps. It would be nice if one could choose how many, like perhaps "only keep the last 5 backups" and delete the old ones.

Thanks for your hard work!

**Comment by glimmerwell** [comment]
-No, the browser doesn't auto-start with the system. I can configure it to do that if that helps.
-I may have misspoke. I re-enabled auto backup. The backup folder auto open every time I start the browser. Maybe it just does it once a day? I have option set: create backup every 1 day.

**Comment by glimmerwell** [comment]
Ok, update: it is making the back up a few times a day. It is putting the backup in what I think is the default folder, and overwriting the backup file from earlier in the day. So there is only one file (at a time) a day. Each time it makes an audio noise and opens the backup folder. Pretty annoying. But maybe it is just something with my setup. Great program, though!

**Comment by Drive4ik** [comment]
> Ok, update: it is making the back up a few times a day. It is putting the backup in what I think is the default folder, and overwriting the backup file from earlier in the day. So there is only one file (at a time) a day.

Yes that's right. If there is an interval of one day, then I change the interval to 2 hours, and enable overwriting the file. This is done so that when FF crashes, you can restore the nearest data, and not the data that was at the beginning of the working day.
Try to run FF with this option:
`C:\...\firefox.exe -p`
You will see a window with profiles, create a new profile, install the addon there and set the interval to 1 hour and wait.
Create a couple of groups, and a couple of tabs in each of the groups. If there are no groups, or they will all be empty, the addon will not backup, in order to avoid overwriting a valid backup to a damaged one.
If the folder will be shown, then you need to look either in the FF settings or in the settings of explorer itself. For example, he has a setting to restore closed windows when the system is turned off, so the setting should be next.

**Comment by grahamperrin** [comment]
Some time ago, Simple Tab Groups gained the ability to use a shortcut for an on-demand backup: 

https://github.com/Drive4ik/simple-tab-groups/issues/595#issuecomment-782605967
